agcaballero/walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -33,54 +33,122 @@
    */ 
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
     Collection<String> attendees = request.getAttendees();
-    ArrayList<TimeRange> times = new ArrayList<TimeRange>();
-     // minutes of day is + 1 to account for first and last minute of day
-    boolean[] minutes = new boolean[MINUTES_IN_DAY + 1];
-
-    for(int i = 0; i < minutes.length; i++) {
-      minutes[i] = true;
+    Collection<String> optionalAttendees = request.getOptionalAttendees();
+    
+    // are there any mandatory employees?
+    boolean mandatory = false;
+    if(attendees.size() > 0) {
+      mandatory = true;
+    }
```

Comment:
I think these lines could be condensed to:

```
boolean mandatory = (attendees.size() > 0);
```

Also applies below.

Reply:
I agree. I was about to make that change myself.

Reply:
Actually, I ended up taking this variable out entirely (as I realized a way it wasn't actually necessary). While I do agree with this and the below suggestions regarding these variables, they are currently deleted. Else, I would have adopted these changes.

Reply:
Done